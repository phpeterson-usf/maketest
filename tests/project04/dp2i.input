-h $(repo_dir)/dp2i.hex $(TESTS_DIR)/dp2i.s
