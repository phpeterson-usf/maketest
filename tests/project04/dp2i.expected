v2.0 raw
e3a0002a
e12fff1e
